.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build and Release
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v2
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: sudo apt-get -qq update
+      - run: sudo apt-get install -y oathtool
+      - run: npm install
+      - run: npm run test:unit:local
+      - name: Upload coverage report to CodeCov
+        uses: codecov/codecov-action@v1
+        with:
+          file: coverage/clover.xml # optional
+          fail_ci_if_error: true # optional (default = false)
+          verbose: true
+      - name: Publish to NPM
+          uses: pascalgn/npm-publish-action@1.3.4
+          with: # All of theses inputs are optional
```

Comment:
Does this comment mean the command works without the inputs?

Reply:
Yes, it does. the comment was from the example code I copy/pasted showing how to use the plugin. There are defaults for all these inputs and we are overriding them for our own usecase.